README.md

```diff
@@ -36,6 +39,7 @@ export BUILD_HARNESS_EXTENSIONS_PATH ?= $(BUILD_HARNESS_PATH)/build-harness-exte
   kubeval/validate                    Validate manifests
   opa/clone-policy                    Git clone policies (requires OPA_POLICY_REPO argument)
   opa/conftest                        Validate manifests
+  pipenv/lock                         Lock Pipenv dependencies
```

Comment:
Do you want to add a poetry lock command?

Reply:
it is already there

Reply:
Even better!